code/modules/cm_marines/orbital_cannon.dm

```diff
@@ -533,7 +533,7 @@ var/obj/machinery/ship_rail_gun/almayer_rail_gun
 	bound_height = 64
 	bound_y = 64
 	unacidable = TRUE
-	use_power = 1
+	use_power = TRUE
```

Comment:
`use_power` should rather be `IDLE_POWER_USE` than `TRUE`, as there's three possible states.